django-stubs/db/models/query.pyi

```diff
@@ -154,6 +154,9 @@ class _ValuesQuerySet(Generic[_T, _Row], Collection[_Row], Reversible[_Row], Que
     def distinct(self, *field_names: Any) ->  _ValuesQuerySet[_T, _Row]: ... # type: ignore
     def order_by(self, *field_names: Any) -> _ValuesQuerySet[_T, _Row]: ... # type: ignore
     def all(self) -> _ValuesQuerySet[_T, _Row]: ... # type: ignore
+    def annotate(self, *args: Any, **kwargs: Any) -> _ValuesQuerySet[_T, Any]: ...
```

Comment:
Please, see https://github.com/typeddjango/django-stubs/pull/398

We are working on fixing `Any` there